english/animal/animal.md

```diff
@@ -3,4 +3,9 @@ My worst fears have come true.  My cat is crying out for food.
 Even if it really start to drop tears or weep blood, it's the same to me ... no food.
 The hilarious thing is that i start thinking about making it my own meal.
 I remember my last meal, oh it was a dead dog 3 days ago.
-We are stepping to the verge of eating each other, human flesh oh what does it taste.
\ No newline at end of file
+We are stepping to the verge of eating each other, human flesh oh what does it taste.
+
+Halfway eating, you notice that something is wrong. Where is the meatiness? The tinge
+of iron? Why is it so... sweet? 
+
+To your horror, your dog, your cat, and you yourself are [marshmallow.](english/marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.